applications/zencode-for-substrate.md

```diff
@@ -0,0 +1,129 @@
+
+
+
+## Team :busts_in_silhouette:
+(LinkedIn profile links in table)
+
+| Tech | Non-Tech | Support |
+| :--- | :--- | :--- |
+| [Robert Miller](https://www.linkedin.com/in/robert-m-2b629335/) - CTO | [Lorenzo Ferrari](https://www.linkedin.com/in/lorenzo-ferrari-669409125/) - CEO | [Dyne](http://dyne.org/) |
+| [Chris Kraynik](https://www.linkedin.com/in/tidus/) - DevOps | [Leonie Forsman](https://www.linkedin.com/in/leonieforsman/) - COO | [dittofi](https://dittofi.com/) |
+| [Sohaib Ezzahir](https://www.linkedin.com/in/sohaibezzahir/) - Front End | | |
+
+### Contact
+* **Name:** Lorenzo Ferrari, CEO
+* **Email:** lorenzo@cryptoarena.org
+* **Web site:** https://cryptoarena.org/
+
+### Legal Structure 
+* **Registered Address:** Stoomgemaalstraat 9, 1069WS North Holland, Amsterdam, NL
+* **Registered Legal Entity:** CryptoArena
+
+### Team's experience
+* https://www.linkedin.com/rabm2k7
+* https://www.linkedin.com/irvinex	
+
+### Team Code Repos
+* https://github.com/CryptoArenaSDEX/Zencode-for-Substrate
+
+
+
+## Development Roadmap :nut_and_bolt:
+
+### Overview
+* **Total Estimated Duration:** 1 months
+* **Full-time equivalent (FTE):** 4.5 FTE
+* **Total Costs:** $30,000 USD
+
+
+
+### Milestone 1 — Zencode for Substrate
+
+CMD LINE -> API -> Smart Contract -> Substrate / Polkadot
+
+* **Estimated Duration:** 1 month
+* **FTE:** 4.5
+* **Costs:** $30,000 USD
+
+| Number | Deliverable | Specification |
+| :--- | :--- | :--- |
+| 0a. | License | GNU GPL V3 |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a Substrate user can use it. |
+| 0c. | Testing Guide | The code will have unit test coverage (min. 70%) to ensure functionality and robustness. In the guide, we will describe how to run these tests. | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. |
+| 1. | Node.js CMD Line Interface | We will create a Node.js command line interface that will integrate with our API-based Zencode smart contract engine (2). |
+| 2. | Node.js API Zencode Smart Contract Engine | We will create an extensible API that interprets input, processes it inline with the chosen blockchain smart contract "recipe", and provides the output prepared for transmission to the blockchain. By "extensible", we mean that Zencode smart contract recipes for blockchains other than Polkadot can be created and integrated easily. |
+| 3. | Zencode Smart Contract Recipe: Polkadot | We will create a smart contract recipe for our API that integrates with the Polkadot blockchain. This Zencode smart contract will be capable of interacting with all public functions of the Polkadot blockchain. |
+| 4. | Dockerfile | We will provide a Dockerfile to demonstrate the full functionalities of our modules. |
+
+
+
+## Future Plans
+
+### Milestone 2 - Zenchain Substrate Pallet
```

Comment:
Just so we are on the same page: this application is for Milestone 1 only, correct? It's good to know that you have concrete plans for the future, though. :)

Reply:
Correct, just Milestone 1 for now, and we'll apply for follow ups in due time. 
And thank you, it'll be cool!

Reply:
Hi @semuelle , can I ask you how long should we expect the review process to take? 
We're eager & ready.

Reply:
Hi @CryptoArenaCEO, if you don't hear from one of us this week, please complain. 
Generally, an application needs three approvals, meaning three evaluators need to read, understand and approve the application. We have no strict timelines or deadlines on these; depending on the number of questions and complexity of the application, it might take a few days to several weeks.